src/lightkurve/targetpixelfile.py

```diff
@@ -2045,7 +2051,60 @@ def plot_pixels(
 
         return ax
 
+    @property
+    def skycatalog(self):
+        """Function that returns an astropy table of sources with the proper motion correction applied
 
+        Returns:
+        ------
+        catalog : astropy.table.Table
+        Returns an astropy table with ID, RA and Dec coordinates corrected for propermotion, and Mag
+        """
+
+        if self.mission.lower() == "kepler":
+            catalog_name = "kic"
+            radius = 4 * (np.max([self.shape[1], self.shape[2]]) + 10) / 2 * u.arcsec
+        elif self.mission.lower() == "k2":
+            catalog_name = "epic"
+            radius = 4 * (np.max([self.shape[1], self.shape[2]]) + 10) / 2 * u.arcsec
+        elif self.mission.lower() == "tess":
+            catalog_name = "tic"
+            radius = 21 * (np.max([self.shape[1], self.shape[2]]) + 10) / 2 * u.arcsec
+        else:
+            raise ValueError("Must pass a valid Target Pixel File object.")
+
+        catalog = query_skycatalog(
+            coord=SkyCoord(self.ra, self.dec, unit="deg"),
```

Comment:
Should the target coordinate (`self.ra, self.dec`) be proper motion  corrected before sending the query out?

Reply:
This is a very good point. We want all objects over a certain brightness within the TPF to be listed in the catalog. The input coordinate must allow for this. If the J2000 R.A and Dec from the TPF is used or even the values corrected for proper motion - the list of returned objects may not cover the TPF. As such i have now adjusted the code such that when a TPF is used the central coordinates of the TPF are used within the catalog search. 

We cannot use the corrected co-ordinates as suggested above since it is from this catalog search we get the proper motion values, and again this would not be beneficial in returning all objects within the TPF FOV.

Reply:
Good point that TPF center should be used for catalog search.